schemas/api_schema.js

```diff
@@ -134,9 +141,11 @@ type Mutation {
                 info
             ) => {
                 check_sessions_count(count);
+                const now = new Date();
                 return db.Session.findAll({
-                    where: { user_id },
-                    limit: count
+                    where: { user_id, end_date: { [Op.gt]: now } },
```

Comment:
IMO should not be filtered in this specific query

Reply:
changed